### ISSUE TITLE
Update dgm-gravferden API to not accept org and app as parameters

### ISSUE THREAD

**Issue by thechrrust** [Altinn/digital-gravferd#40]
### Description

Currently, the EventsController in dgm-gravferden looks like this:

```csharp
[Route("{org}/{app}")]
[ApiController]
public class EventsController : ControllerBase
{
    ....

    [HttpPost("api/[controller]/process")]
    public async Task<IActionResult> ProcessEvent([FromBody] CloudEvent cloudEvent)
    {
         ....
    }

}
```

Here, org and app are configured as settable parameters that the actions can take as input. This is not the case, it should be specific to ttd and dgm-gravferden at the current time. When we go over to Statsforvalteren org, Statsforvalteren should be the org. 

It doesn't need to be parameterized, and can be hardcoded.

### Tasks

- [ ] Remove parameterization of org and app in route - it should be hardcoded

### Acceptance Criterias

It should no longer be possible to change org and app in the route

**Comment by thechrrust** [comment]
Should be merged with https://github.com/Altinn/digital-gravferd/issues/34

**Comment by thechrrust** [comment]
Merged into #34
